src/openrct2/sprites.h

```diff
@@ -1293,6 +1293,9 @@ enum
     SPR_RIDE_DESIGN_PREVIEW_CLASSIC_WOODEN_ROLLER_COASTER_TRACK,
     SPR_RIDE_DESIGN_PREVIEW_CLASSIC_WOODEN_ROLLER_COASTER_SUPPORTS,
 
+    SPR_G2_BM_INVERT_BOOSTER_1,
+    SPR_G2_BM_INVERT_BOOSTER_2,
+
     SPR_G2_LIM_LAUNCHED_TRACK_BEGIN = SPR_RIDE_DESIGN_PREVIEW_CLASSIC_WOODEN_ROLLER_COASTER_SUPPORTS + 1,
```

Comment:
This breaks the LIM launched coaster’s sprites. Thankfully, the g2 warning caught this. I’ll push a fix.